### ISSUE TITLE
Any up-to-date alternatives?

### ISSUE THREAD

**Issue by dm17** [artetxem/mitzuli#46]
Looking for any offline translators for Android that do not require the Google Play Store. I thought this might be a good place to check :)

**Comment by qyczq** [comment]
have you found any?

**Comment by dm17** [comment]
Nope, still a huge need! Especially ones that can do speech to text across languages.

**Comment by qyczq** [comment]
look what i have found https://github.com/niedev/RTranslator
